.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature request
+description: Suggest a new idea for FastTrackML
+title: "[FEATURE REQUEST]: <title>"
+labels: [ "feature" ]
+
+body:
+  - type: textarea
+    id: related-problem
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+    validations:
+      required: true
+
+  - type: textarea
+    id: describe-solution
+    attributes:
```

Comment:
Likewise, the new feature isn't necessary a "solution to a problem" but can just be something new. I'd think this could be more "describe the feature you'd like to see" -- and it should be the first field.